Tasks/DownloadFileShareArtifactsV0/task.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "E3CF3806-AD30-4EC4-8F1E-8ECD98771AA0",
+  "name": "DownloadFileShareArtifacts",
+  "friendlyName": "Download FileShare Artifacts",
+  "description": "Download FileShare Artifacts",
+  "helpMarkDown": "",
```

Comment:
All tasks should have a fwlink to docs page for the task. We want to show yaml snippets along with some examples on the doc page.

Reply:
not sure how doc is generated, I don't see docs checked in with tasks. Is there some other repo for docs?